iree/compiler/Conversion/HLOToHLO/Passes.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//===- Passes.h - Codegen pass to convert from XLA-HLO to XLA-HLO ---------===//
+//
+// IREE specific passes used in the XLA-HLO to XLA-HLO
```

Comment:
Could answer some of my questions about "hlo to hlo" "conversion" in this block comment. As it is now, this comment is sparse on details.

Reply:
Flushed out a little more detail. In general this should be any passes to validate towards our ingested HLO.